data/json/items/tool/cooking.json

```
@@ -685,6 +685,13 @@
         "max_contains_volume": "20 L",
         "max_contains_weight": "20 kg",
         "item_restriction": [ "medium_battery_cell", "medium_plus_battery_cell", "medium_atomic_battery_cell", "medium_disposable_cell" ]
+      },
+      {
+        "pocket_type": "CONTAINER",
```

```suggestion
        "pocket_type": "CONTAINER",
        "watertight": true,
        "rigid": true,
```
Should be watertight and rigid.